multi_physics/QED/python_bindings/CMakeLists.txt

```
@@ -0,0 +1,64 @@
+set(name pxr_qed)
+
+find_package(Python COMPONENTS Interpreter Development REQUIRED)
+find_package(pybind11 CONFIG REQUIRED)
+
+pybind11_add_module(${name} "${name}.cpp")
+
+target_link_libraries(${name} PRIVATE PXRMP_QED)
+
+option(
+    PXRQEDPY_DOUBLE_PRECISION
+    "Compile pxr_qed python module in double precision" ON)
+
+if(PXRQEDPY_DOUBLE_PRECISION)
+    target_compile_definitions(${name} PRIVATE PXRQEDPY_DOUBLE_PRECISION=1)
+endif()
+
+set(PXRQEDPY_UNITS_VALUES SI NORM_OMEGA NORM_LAMBDA HEAVISIDE_LORENTZ)
+set(PXRQEDPY_UNITS SI CACHE STRING "Unit system for the pxr_qed python module (SI/NORM_OMEGA/NORM_LAMBDA/HEAVISIDE_LORENTZ)")
+set_property(CACHE PXRQEDPY_UNITS PROPERTY STRINGS ${PXRQEDPY_UNITS_VALUES})
+
+if(NOT PXRQEDPY_UNITS IN_LIST PXRQEDPY_UNITS_VALUES)
+    message(FATAL_ERROR "PXRQEDPY_UNITS (${PXRQEDPY_UNITS}) must be one of ${PXRQEDPY_UNITS_VALUES}")
+endif()
+
+if(PXRQEDPY_UNITS STREQUAL "SI")
+    target_compile_definitions(${name} PRIVATE PXRQEDPY_SI=1)
+elseif(PXRQEDPY_UNITS STREQUAL "NORM_OMEGA")
+    target_compile_definitions(${name} PRIVATE PXRQEDPY_NORM_OMEGA=1)
+elseif(PXRQEDPY_UNITS STREQUAL "NORM_LAMBDA")
+    target_compile_definitions(${name} PRIVATE PXRQEDPY_NORM_LAMBDA=1)
+elseif(PXRQEDPY_UNITS STREQUAL "HEAVISIDE_LORENTZ")
+    target_compile_definitions(${name} PRIVATE PXRQEDPY_HEAVISIDE_LORENTZ=1)
+else()
+    message(FATAL_ERROR "PXRQEDPY_UNITS is incorrectly defined")
+endif()
+
+set_target_properties(${name} PROPERTIES
+                              INTERPROCEDURAL_OPTIMIZATION ON)
+
```

Rather use this:
```suggestion
target_link_libraries(${name} PRIVATE pybind11::module pybind11::lto pybind11::windows_extras)
```
But `add_python_module` does include LTO already: https://pybind11.readthedocs.io/en/stable/compiling.html#advanced-interface-library-targets

So you can remove this fully.
```suggestion
```